simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -26,4 +46,87 @@ const ShareQueue = (props) => {
   );
 };
 
+const ShareQueue = (props) => {
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
+
+  const quote = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
+  const link = `${window.location.origin}/j/${props.queueName}`;
```

Comment:
nit: Define link first and use it inside quote.

Reply:
done